submissions/kris-almort/a-tiny-js-world/index.js

```diff
@@ -72,18 +72,23 @@ const catWoman = {
 };
 
 function preparePrint(obj) {
+  const { species, name, gender, legs, saying } = obj;
+  const hands = obj.hasOwnProperty("hands") ? `hands: ${obj.hands};` : "";
+  const friends =
```

Comment:
Same: change naming, use destructuring.

You can use optional chaining here for check. [Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

`friends?.length ? doSmth : doSmthElse`

Reply:
fixed :)